ivy_tests/test_ivy/test_frontends/test_numpy/test_logic/test_logical_operations.py

```diff
@@ -1,3 +1,4 @@
+# for review
```

Comment:
small detail, but `(ivy_np.bool,) + (None,)` should be replaced with `(ivy_np.bool, None)`